samples/apps/helloworld/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.11-alpine AS build
+
+COPY . src
+WORKDIR src
+RUN GO111MODULE=on CGO_ENABLED=0 go build -o /bin/hello
+
+FROM scratch
+COPY --from=build /bin/hello /bin/hello
+
+ENV PORT 8080
```

Comment:
nit: Are we sure we should have the PORT and EXPOSE here? I thought we didn't support this and required the app to still listen for what Knative sets.

Reply:
I added a comment here indicating you don't need these in kf, but they are there if you wanted to try just running the docker image yourself.